src/mixins/filing-template-mixin.ts

```diff
@@ -432,8 +432,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           identifier: this.getBusinessId
         },
         contactPoint: {
-          email: this.getBusinessContact.email,
-          phone: this.getBusinessContact.phone,
+          email: this.getBusinessContact.email || 'unknown@example.com',
+          phone: this.getBusinessContact.phone || '',
```

Comment:
@argush3 FYI

Reply:
I'm changing this:
- if there's no email or phone, don't include Contact Point object
- if there's no phone extension, don't include it

Wait for next commit.

Reply:
I have no idea what to put here if email or phone are empty. Waiting to hear back from Argus with more info.

Maybe this?
```
contactPoint: {
  email: this.getBusinessContact.email || undefined, // don't include if empty
  phone: this.getBusinessContact.phone || undefined, // don't include if empty
  extension: +this.getBusinessContact.extension || undefined // don't include if empty
},
```